modules/core/src/main/scala/org/tessellation/cli/snapshot.scala

```diff
@@ -0,0 +1,29 @@
+package org.tessellation.cli
+
+import cats.syntax.contravariantSemigroupal._
+
+import org.tessellation.config.types.SnapshotConfig
+import org.tessellation.ext.decline.decline._
+
+import com.monovore.decline._
+import com.monovore.decline.refined._
+import eu.timepit.refined.types.numeric.PosLong
+import fs2.io.file.Path
+
+object snapshot {
+
+  val heightIntervalOpts: Opts[PosLong] = Opts
+    .env[PosLong]("CL_SNAPSHOT_HEIGHT_INTERVAL", help = "Height interval between snapshots")
+    .withDefault(PosLong(2))
+
+  val heightDelayIntervalOpts: Opts[PosLong] = Opts
+    .env[PosLong]("CL_SNAPSHOT_HEIGHT_DELAY_INTERVAL", help = "Height delay of snapshot trigger")
+    .withDefault(PosLong(2))
+
+  val storedSnapshotPath: Opts[Path] = Opts
```

Comment:
I wonder if we want these to be loaded from envs or just hardcoded in an object?